manager/assets/modext/core/modx.layout.js

```diff
@@ -465,10 +475,13 @@ Ext.extend(MODx.Layout, Ext.Viewport, {
                     }
                 }
             });
-            buttons[i].onclick = function(e) {
+            var _this = this;
+            var openSubnav =
```

Comment:
This seems to be invalid :)

Reply:
@theboxer i also removed this bug 4be7479a8b12f52d193169d76f2c77e8f9c80bb6 ✅